mayland-blocks/block-templates/single.html

```diff
@@ -12,7 +14,13 @@
 <div style="height:16px" aria-hidden="true" class="wp-block-spacer"></div>
 <!-- /wp:spacer -->
 
-<!-- wp:post-content /-->
+</div>
+<!-- /wp:group -->
+
+<!-- wp:post-content {"layout":{"inherit":true}} /-->
+
+<!-- wp:group {"layout":{"inherit":true}} -->
+<div class="wp-block-group">
 
 <!-- wp:spacer {"height":16px} -->
 <div style="height:16px" aria-hidden="true" class="wp-block-spacer"></div>
```

Comment:
Could be addressed separately, but are spacer blocks the right way to go for adding this kind of margin in the template? They are nice because they give users control in a FSE context, but I also saw some discussion about this yesterday in slack that suggested we use margin/padding instead.

Reply:
Yeah, and with the coming margins to the group block that might be even better. I will address it on a separate PR since it's not alignment related, thanks!